ion/include/ion/python_heap.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
```

Comment:
Prefere using
```suggestion
#ifndef ION_PYTHON_HEAP_H
#define ION_PYTHON_HEAP_H
```
and add ```cpp
#endif``` at the end of the file